config/initializers/friendly_id.rb

```diff
@@ -44,14 +44,14 @@
   # performance because it will avoid Rails-internal code that makes runtime
   # calls to `Module.extend`.
   #
-  # config.use :finders
+  config.use :finders
```

Comment:
Don't use finders module, it does have side effects

Reply:
Okay, I removed it and checked that it is working fine...